26-register_publication/1633-creator_selects_reference_type_other_publication_and_selects_subtype_other_publication.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Creator selects Reference type Other publication and selects subtype Other publication
+
+    @1633
+    Scenario: Creator selects Reference type "Other publication" and selects subtype "Other publication"
+        Given Creator begins registering a Publication in the Wizard
+        When they navigate to the Reference tab
+        And they select the Reference type "Other publication"
+        And they select the subtype "Other publication"
+        Then they see fields
+            | Original version            |
+            | Search box for Published in |
+            | Search box for Publisher    |
```

Comment:
hva er forskjellen på `Published in` og `Publisher`?

Reply:
Ser ut som `Published in` er spesielt for Feature article og other publication: https://unitdir.sharepoint.com/:x:/r/sites/prosjekt-51110085001/_layouts/15/Doc.aspx?sourcedoc=%7B1BABA128-0B63-4E57-A431-E80D6485771B%7D&file=Ressurstyper_med_felter.xlsx&wdOrigin=OFFICECOM-WEB.MAIN.REC&ct=1602485730278&action=default&mobileredirect=true